src/utils/boost.ts

```diff
@@ -30,7 +30,7 @@ export const getTimeFactor = (days: number) => {
   }
 
   if (days <= 47) {
-    return 1 - 0.0106383 * days
+    return 1 - 0.010638 * days
```

Comment:
I adjusted this because the function was going down until the start of season one date, and then increasing slightly before decreasing again towards the end of season 2. The color changes at the midpoint so it was going green/red/green/red around the midpoint

Reply:
I commented this in the spec document. If we adjust this here our computations during the pre season might be slightly higher than the boosts actually are. 
The boost functions will be changed anyways. So I'd say lets wait for that to see if this change is still relevant.